catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-common/src/main/java/org/codice/ddf/spatial/ogc/wfs/v2_0_0/catalog/common/Wfs20Constants.java

```diff
@@ -111,27 +111,31 @@
     public String getValue() {
       return value;
     }
+
+    @Override
+    public String toString() {
+      return value;
+    }
   }
 
-  public static enum CONFORMANCE_CONSTRAINTS {
-    ImplementsQuery("ImplementsQuery"),
-    ImplementsAdHocQuery("ImplementsAdHocQuery"),
-    ImplementsFunctions("ImplementsFunctions"),
-    ImplementsResourceld("ImplementsResourceld"),
-    ImplementsMinStandardFilter("ImplementsMinStandardFilter"),
-    ImplementsStandardFilter("ImplementsStandardFilter"),
-    ImplementsMinSpatialFilter("ImplementsMinSpatialFilter"),
-    ImplementsSpatialFilter("ImplementsSpatialFilter"),
-    ImplementsMinTemporalFilter("ImplementsMinTemporalFilter"),
-    ImplementsTemporalFilter("ImplementsTemporalFilter"),
-    ImplementsVersionNav("ImplementsVersionNav"),
-    ImplementsSorting("ImplementsSorting"),
-    ImplementsExtendedOperators("ImplementsExtendedOperators"),
-    ImplementsMinimumXPath("ImplementsMinimumXPath"),
-    ImplementsSchemaElementFunc("ImplementsSchemaElementFunc"),
-    ImplementsResourceId("ImplementsResourceId");
-
-    private String value;
+  public enum CONFORMANCE_CONSTRAINTS {
+    IMPLEMENTS_QUERY("ImplementsQuery"),
+    IMPLEMENTS_AD_HOC_QUERY("ImplementsAdHocQuery"),
+    IMPLEMENTS_FUNCTIONS("ImplementsFunctions"),
+    IMPLEMENTS_RESOURCE_ID("ImplementsResourceId"),
```

Comment:
Notice that I removed the last enum and changed the value here. I think there was a typo, so I updated this from `ImplementsResourceld` to `ImplementsResourceId`.